packages/graph/hash_graph/lib/graph/src/ontology/link_type.rs

```diff
@@ -27,6 +27,62 @@ impl QueryRecord for LinkType {
     type Path<'q> = LinkTypeQueryPath;
 }
 
+impl RecordPath for LinkTypeQueryPath {
+    fn expected_type(&self) -> ParameterField {
+        match self {
+            Self::OwnedById | Self::CreatedById | Self::UpdatedById => ParameterField {
+                parameter_type: ParameterType::Uuid,
+                optional: false,
+            },
+            Self::RemovedById => ParameterField {
+                parameter_type: ParameterType::Uuid,
+                optional: true,
+            },
+            Self::BaseUri => ParameterField {
+                parameter_type: ParameterType::BaseUri,
+                optional: false,
+            },
+            Self::VersionedUri => ParameterField {
+                parameter_type: ParameterType::VersionedUri,
+                optional: false,
+            },
+            Self::Version => ParameterField {
+                parameter_type: ParameterType::UnsignedInteger,
+                optional: false,
+            },
+            Self::Title => ParameterField {
+                parameter_type: ParameterType::Timestamp,
```

Comment:
This should be text, right? (and the one below) Also I'm not sure `Description` _is_ optional for link types, might want to check the schema on that.

This helps emphasise my guiding question above, concerned about the maintainability here and discoverability of bugs/edge-cases

Reply:
Thanks, fixed in b5edf96d3bf7f9964d1af566176eeeb77b05a125